packages/table/src/table.tsx

```diff
@@ -530,14 +561,34 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private bodyCellRenderer = (rowIndex: number, columnIndex: number) => {
-        return this.getColumnProps(columnIndex).renderCell(rowIndex, columnIndex);
+        const columnProps = this.getColumnProps(columnIndex);
+        const cell = columnProps.renderCell(rowIndex, columnIndex);
+        const { loading: cellLoading } = cell.props;
```

Comment:
_whoa_ i did not know this was valid syntax! since you're only deconstructing one prop, i'd just do the simple legible `const cellLoading = cell.props.loading`.